TRRandomizerView/Utilities/ProcessUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Diagnostics;
+using System.IO;
+
+namespace TRRandomizerView.Utilities;
+
+public class ProcessUtils
+{
+    public static void OpenFile(string fileName, string arguments = null)
+    {
+        fileName = Path.GetFullPath(fileName);
+        Process.Start(new ProcessStartInfo
+        {
+            FileName = fileName,
+            Arguments = arguments,
+            UseShellExecute = true,
+            WorkingDirectory = Path.GetDirectoryName(fileName)
+        });
+    }
+
+    public static void OpenFolder(string folder)
+    {
+        Process.Start(new ProcessStartInfo
+        {
+            FileName = "explorer.exe",
+            Arguments = Path.GetFullPath(folder),
+            UseShellExecute = true
+        });
+    }
+
+    public static void OpenURL(string url)
+    {
+        Process.Start(new ProcessStartInfo
+        {
+            FileName = url,
+            UseShellExecute = true
+        });
+    }
+}
```

Comment:
Do you think it'd be useful to have similar class for file dialogs?

Reply:
Good idea, a bit tidier and useful if we need to change something again in the future. New commit in with the update.